config/organization/clusterworkspacetype-use-clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:kcp:universal-clusterworkspacetype-use
+rules:
+- apiGroups: ["tenancy.kcp.dev"]
+  resources: ["clusterworkspacetypes"]
+  resourceNames: ["universal"]
+  verbs: ["use"]
```

Comment:
since this is used only in the `create` case, it seems a bit counter-intuitive to use the verb `use` here. do we want to use that verb for other cases as well?

Reply:
I agree but I don't have any alternative suggestions at the moment. Let's maybe file an issue to re-review/follow-up in the future?

Reply:
alternative: instantiate

Reply:
created https://github.com/kcp-dev/kcp/issues/617